apps/docs/src/pages/tabs.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Tabs
+description: Sample Tabs documentation
+layout: ../layouts/MainLayout.astro
+---
+
+import { Tab } from "@selleo/core/src/Tabs";
+
+import Preview from "../components/Preview.astro";
+import Description from "../components/Description.astro";
+
+<Description>
+  Below you can find a tabs component styled with tailwindcss.
+</Description>
+
+<Preview component={Tab} name="Default tab" active={false} />
+
+<Preview component={Tab} name="Active tab" active={true} />
+
+<Preview
+  component={Tab}
+  name="Tab with icon"
+  active={false}
+  icon={<span>x</span>}
+/>
```

Comment:
please add a row of tabs view like on the top of the design
multiple tabs with one of them selected

Reply:
Done